Web/Edubase.Web.UI/Views/Guidance/ReadyToDownload.cshtml

```diff
@@ -0,0 +1,49 @@
+
+@model Edubase.Web.UI.Models.Guidance.GuidanceLaNameCodeViewModel
+@{
+    ViewBag.hideLogo = true;
+}
+@section Head
+{
+    <meta name="robots" content="noindex,nofollow" />
+}
+@section BreadCrumbs
+{
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <div class="govuk-breadcrumbs">
+                <ol class="govuk-breadcrumbs__list">
+                    <li class="govuk-breadcrumbs__list-item">
+                        @Html.ActionLink("Home", "Index", "Home", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })
+                    </li>
+                    <li class="govuk-breadcrumbs__list-item">
+                        @Html.ActionLink("Help", "Help", "Home", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })
+                    </li>
+                    <li class="govuk-breadcrumbs__list-item">
+                        @Html.ActionLink("Local authority name and associated codes", "LaNameCodes", "Guidance", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })
+                    </li>
+                    <li class="govuk-breadcrumbs__list-item">
+                        @Html.ActionLink("Select data table", "LaNameCodes/DataTables", "Guidance", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })
+                    </li>
+                    <li class="govuk-breadcrumbs__list-item">
+                        @Html.ActionLink("Select format", "LaNameCodes/DataTables/SelectFormat", "Guidance", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })
```

Comment:
Could use "LaNameCodesSelectFormat" instead of "LaNameCodes/DataTables/SelectFormat"

I am getting a warning on this in Rider, which I why I looked here